docs/howto/features/cryptnono.md

```diff
@@ -0,0 +1,176 @@
+   ```
+
+   This tells us that the name of the process, as well as why it was killed.
+
+   ```{note}
+   We will eventually be able to see the [pod & namespace](https://github.com/cryptnono/cryptnono/issues/30) information
+   of killed processes as well.
+   ```
+
+   ```{tip}
+   To make the JSON easier to read, you can pipe the logs command to `jq`.
+   ```
+
+## Regenerating list of banned strings
+
+Periodically, we will have to regenerate the list of banned strings to tune `cryptnono`,
+by running the following command:
+
+```bash
+deployer generate cryptnono-secret-config
+```
+
+This will update the file `helm-charts/support/enc-cryptnono.secret.values.yaml` and re-encrypt it.
+
+## Working on the banned strings generator
+
+The banned strings generator is a fairly simple python script, present in `deployer/commands/generate/cryptnono_config/enc-blocklist-generator.secret.py`.
+It's unencrypted and loaded by code in `deployer/commands/generate/cryptnono_config/__init__.py`. There is inline
+documentation in `encrypted_secret_blocklist.py`, but how does one hack on it?
+
+1. Unencrypt the file with `sops`
+
+   ```bash
+    sops --decrypt deployer/commands/generate/cryptnono_config/enc-blocklist-generator.secret.py > deployer/commands/generate/cryptnono_config/blocklist-generator.secret.py
+   ```
+
+   This will create `deployer/commands/generate/cryptnono_config/blocklist-generator.secret.py` (which is in `.gitignore` and hence
+   can not be committed accidentally) with the unencrypted code.
+
+   ```{note}
+
+   Because this file is in `.gitignore`, your IDE may not show it to when you search for files by default!
+   ```
+
+2. Work on the code as you wish - it's just a regular python file.
+
+3. Re-encrypt it with `sops`
+
+   ```bash
+    sops --encrypt deployer/commands/generate/cryptnono_config/blocklist-generator.secret.py > deployer/commands/generate/cryptnono_config/enc-blocklist-generator.secret.py
+   ```
+
+4. Run `deployer generate cryptnono-secret-config` to test.
+
+## Right to Replicate considerations
+
+2i2c's [Right to Replicate](https://2i2c.org/right-to-replicate/) guarantees that communities can leave
+with their hubs whenever they please, without any secret sauce. Cryptnono has two pieces of secret info
+here:
+
+1. The script to generate the secret config.
+2. The secret config itself.
+
+If a community wishes to leave and needs the config, we can make sure they know what the config is and
+how to keep it updated - very similar to how we would handle passing on CILogon credentials or similar to
+them. Since none of the code for cryptnono itself is secret, this does not conflict with right to replicate.
+
+## Future work
+
+`cryptnono` also exposes prometheus metrics about processes it has killed. We currently do not collect these,
+but we should enable collection so we can
```

Comment:
I think it misses a few closing words :D

Reply:
Ah yep! I've actually enabled collection of these now, and updated this section to say that we just don't have a grafana board yet.